webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumbs.scss

```diff
@@ -2,3 +2,7 @@
   display: inline-flex;
   max-width: 50%;
 }
+
+.breadcrumbs-pf-title li.breacrumb-item-with-icon span {
+  vertical-align: -9px;
```

Comment:
I wonder if we can contribute this to PF, 
or is it only foreman-specific?

Reply:
It’s foreman specific, patternfly’s breadcrumbs does not have icon by design